static/js/default-map-api-key.js

```diff
@@ -13,7 +13,7 @@ export function getDefaultMapApiKey(mapProvider) {
       case 'google':
         return 'AIzaSyB5D45ghF1YMfqTLSzWubmlCN1euBVPhFw';
```

Comment:
should this also be replaced?

Reply:
It's not included as a vulnerabilities, so i'm not sure.

Reply:
it probably should be